test/Microsoft.Health.Fhir.R4.Tests.E2E/Rest/VersionSpecificTests.cs

```diff
@@ -61,5 +61,12 @@ public async Task GivenAResourceThatWasRenamed_WhenSearched_ThenExceptionShouldB
 
             Assert.Equal(HttpStatusCode.NotFound, exception.StatusCode);
         }
+
+        [Fact]
+        public Task GivenR4Server_WhenSupportedVersionIsRequested_ThenCorrectVersionShouldBeReturned()
+        {
+            TestSupportedVersion("4.0.0");
+            return Task.CompletedTask;
```

Comment:
Seems like we can remove this and update the signature to `public void` ?

Reply:
Oops thanks fixed in 54c0730.